app/config/FrontendAppConfig.scala

```diff
@@ -65,8 +65,6 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val timeoutSeconds: String   = configuration.get[String]("session.timeoutSeconds")
   lazy val countdownSeconds: String = configuration.get[String]("session.countdownSeconds")
 
```

Comment:
Do we want to remove betaAuthorizationUrl above as well (and it's associated application.conf stuff)?